include/uv.h

```diff
@@ -274,6 +274,9 @@ UV_EXTERN void uv_loop_delete(uv_loop_t*);
 UV_EXTERN size_t uv_loop_size(void);
 UV_EXTERN int uv_loop_alive(const uv_loop_t* loop);
 UV_EXTERN int uv_loop_configure(uv_loop_t* loop, uv_loop_option option, ...);
+#ifndef _WIN32
+UV_EXTERN int uv_loop_fork(uv_loop_t*);
```

Comment:
Ah, I didn't remember it was called just fork on libev. Let's have it like this for now. We can bikeshed on the name later :-)